streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBRangeIterator.java

```diff
@@ -63,6 +65,9 @@
         final KeyValue<Bytes, byte[]> next = super.makeNext();
         if (next == null) {
             return allDone();
+        } else if (rawLastKey == null) {
+            return next;
```

Comment:
We changed the meaning of `rawLastKey`, need to add some comment for `rawLastKey == null` case.

Reply:
Added a comment